dsl/camel-jbang/camel-jbang-plugin-kubernetes/pom.xml

```diff
@@ -55,6 +55,11 @@
         </dependency>
 
         <!-- OpenShift -->
+        <dependency>
```

Comment:
Is this only used for testing then set test scope

Reply:
It's use run `run` as well

Reply:
where is it needed - it was not there before

Reply:
I looked again ... and actually no, the run code uses the k8s API. Let me see if it actually works with scope test ...

Reply:
done